cli/lib/tasks/verify.js

```diff
@@ -15,7 +15,7 @@ const logger = require('../logger')
 const xvfb = require('../exec/xvfb')
 const state = require('./state')
 
-const VERIFY_TEST_RUNNER_TIMEOUT_MS = +process.env.CYPRESS_VERIFY_TIMEOUT || 30000
+const VERIFY_TEST_RUNNER_TIMEOUT_MS = +util.getEnv('CYPRESS_VERIFY_TIMEOUT') || 30000
```

Comment:
Testing this locally, I'm noticing the capitalization makes a difference on the resolved value when testing with a simple dummy script. 

```package.json
"scripts": {
   "try": "node ./try.js"
}
```

```js
// try.js
const util = require('./cli/lib/util')

console.log(process.env)
console.log(' process.env -.CYPRESS_VERIFY_TIMEOUT', process.env.CYPRESS_VERIFY_TIMEOUT)
console.log('process.env.cypress_verify_timeout -', process.env.cypress_verify_timeout)
console.log('util.getEnv('CYPRESS_VERIFY_TIMEOUT') - ', util.getEnv('CYPRESS_VERIFY_TIMEOUT'))
console.log('util.getEnv('cypress_verify_timeout') -', util.getEnv('cypress_verify_timeout'))
```

.npmrc
```.npmrc
CYPRESS_VERIFY_TIMEOUT=yes
```

Terminal
```bash
> CYPRESS_VERIFY_TIMEOUT='no' npm run try
process.env - {
   ...
  npm_config_cypress_verify_timeout: 'yes',
  CYPRESS_VERIFY_TIMEOUT: 'no',
  ...
}
process.env.CYPRESS_VERIFY_TIMEOUT - no
process.env.cypress_verify_timeout - undefined
util.getEnv('CYPRESS_VERIFY_TIMEOUT') -  no
util.getEnv('cypress_verify_timeout') - yes
```

Reply:
Thanks for the review and the testing @emilyrohrbough.

I had to dig a little deeper to understand how npm handles environment variables. The results are available in [this README](https://github.com/legap/npm-env-variable-examples/blob/main/README.md) in a small test project.

As a conclusion we can see, that npm itself handles the key of an environment variable different, depending on the operating system (I didn't have an OSX available but I would guess it behaves the same as linux).

I also looked up all calls to the `getEnv` method in the cypress codebase and had a look at the tests in [util_spec.js](https://github.com/legap/cypress/blob/develop/cli/test/lib/util_spec.js) and they are always with uppercase keys.

So in my opinion `getEnv` should work on all operating systems when using the key in uppercase and the benefit would be that we can also use the definition for `CYPRESS_VERIFY_TIMEOUT` in `package.json` or `.npmrc`.

Reply:
@legap WOW! You led a very thorough investigation into these differences. I truly appreciate it & this is great information.

You conclusions regarding Mac OS seems correct (I have a Mac). With your findings it appears this accomplishes the intent of how `CYPRESS_VERIFY_TIMEOUT` will be used and accessed for this timeout.